tests/integration/test_unary_functions_2d.py

```diff
@@ -14,74 +14,78 @@
 #
```

Comment:
I guess this test could be gone as well, as there's a more general test now (`test_unary_ufunc.py`).

Reply:
removed in https://github.com/nv-legate/cunumeric/pull/297/commits/7688326a9d9df0f2be961627d1d7eb006cb329ef